### ISSUE TITLE
add simple web server that populates a table based on s3 bucket

### ISSUE THREAD

**Issue by gkiar** [gkiar/clowdr#3]
- example s3 bucket here: `s3://mrneurodata`
- example static page here: http://m2g.io

Desired functionality:
- function should launch a server rather than just a static page
- server must have the possibility of exposing a RESTful API 
- server should/could optionally have credentials required to view the content, as well as a public mode

**Comment by h3ct0rjs** [comment]
It could be implemented in flask ?

**Comment by gkiar** [comment]
I don't see why not @h3ct0rjs! I suspect Django is overkill, but I'm unfamiliar with RESTful API implementations in flask, but perhaps you can teach me 😄

**Comment by h3ct0rjs** [comment]
I will make a few tests using Flask this week and update this issue :smile:

**Comment by Barathwaja** [comment]
Restful is simple bro .. yep flask is easy for beginners. I've knowledge on expressjs ..

**Comment by gkiar** [comment]
@barathwaja I do mean server - the added code should launch a server doing what I described. Really what I "need help with" is finding the time to do this, which @h3ct0rjs seems to be taking care of. More than happy if you want to work together on the S3 crawling aspect, and I can help out with advice since I've done it before.

Please check the [code of conduct](https://github.com/gkiar/clowdr/blob/master/CONTRIBUTING.md) if you want to contribute - it's important :)

**Comment by Barathwaja** [comment]
See NodeJS , Flask are all same case scenarios.. Actually you term those as **Frameworks** not servers..
They were designed in a way that look like server. 
**ExpressJS is a framework for Nodejs like Flask is a framework for Python.**

And I'm happy to join and help. And OOPS just now saw README, since uses python lets go with Flask.

**Comment by h3ct0rjs** [comment]
Hi @Barathwaja Flask not is just for beginners, is a really powerful microservice  that can be used for serving web pages, templating, and managing sessions, besides that is easy to test in my dev enviroment, I also know expressjs, I've done a lot of projects with nodejs too but Just for my curiosity :smile:  I'm learning python and was reading about how flask can be extended using the built-ins of python. Since you seem to have more experience with flask I'd like to work with you @Barathwaja maybe I can learn more with you than trying myself to figure it out a few things, What do you  think @Barathwaja

**Comment by Barathwaja** [comment]
Cool.. @h3ct0rjs ofcourse friend.. will collaborate and work on this. so that v can share each other's knowledge too.. lol y tagged thrice 😂

**Comment by Barathwaja** [comment]
Yes you can do whatever you can using frameworks.. rather than microservice precise would be microframework.. I have worked with normal CRUD in Flask..  to use aws features their SDKs are there so that should not be a problem. Yes I can also learn a bit from you so that v all can share and do..  how do v all connect?
